src/sentry/templates/sentry/organization-auth-provider-settings.html

```diff
@@ -55,7 +55,11 @@ <h3>{{ provider_name }} Authentication</h3>
           {{ field|as_crispy_field }}
           {% if forloop.last %}
             <fieldset class="form-actions">
-              <button class="btn btn-primary" name="op" value="settings">Save Settings</button>
+                {% if disabled %}
+                  <button disabled class="btn btn-primary" name="op" value="settings">Save Settings</button>
+                {% else %}
+                  <button class="btn btn-primary" name="op" value="settings">Save Settings</button>
+                {% endif %}
```

Comment:
The summary specified that we want to prevent removing the auth config. Just to clarify, editing it is also not allowed?

Reply:
Yes, editing and removing. I'll update the summary